<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>642</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QListView" name="listView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_maid_info">
         <attribute name="title">
          <string>Maid info</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_maid_stats">
         <attribute name="title">
          <string>Maid stats</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_maid_work">
         <attribute name="title">
          <string>Maid work</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_yotogi_skills">
         <attribute name="title">
          <string>Yotogi skills</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_player_info">
         <attribute name="title">
          <string>Player Info</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <property name="horizontalSpacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidget">
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="rowCount">
             <number>0</number>
            </property>
            <property name="columnCount">
             <number>3</number>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>69</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Property</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Value</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Lock</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGeneral_tools">
    <property name="title">
     <string>Current save</string>
    </property>
    <addaction name="actionUnlock_all_trohpies"/>
    <addaction name="actionUnlock_all_stock_items"/>
    <addaction name="actionMaximum_credits"/>
    <addaction name="actionMaximum_club_grade_and_evaluation"/>
   </widget>
   <widget class="QMenu" name="menuAll_maids">
    <property name="title">
     <string>All maids</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSelected_maid">
    <property name="title">
     <string>Selected maid</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMisc">
    <property name="title">
     <string>Misc</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuGeneral_tools"/>
   <addaction name="menuAll_maids"/>
   <addaction name="menuSelected_maid"/>
   <addaction name="menuMisc"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUnlock_all_trohpies">
   <property name="text">
    <string>Unlock all trohpies</string>
   </property>
  </action>
  <action name="actionUnlock_all_stock_items">
   <property name="text">
    <string>Unlock all stock items</string>
   </property>
  </action>
  <action name="actionMaximum_credits">
   <property name="text">
    <string>Maximum credits</string>
   </property>
  </action>
  <action name="actionMaximum_club_grade_and_evaluation">
   <property name="text">
    <string>Maximum club grade and evaluation</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
